app/controllers/admin/basic_projects_controller.rb

```diff
@@ -22,54 +22,64 @@ def index
   end
 
   def show
-    @project = BasicProject.find(params[:id])
-    authorize @project
+    @basic_project = BasicProject.find(params[:id])
+    authorize @basic_project
     prep_form_vars
+    @steps = @basic_project.project_steps
+    @calendar_events_url = "/admin/calendar_events?project_id=#{@basic_project.id}"
   end
 
   def new
-    @project = BasicProject.new(division: current_division)
-    authorize @project
+    @basic_project = BasicProject.new(division: current_division)
+    authorize @basic_project
     prep_form_vars
   end
 
   def update
-    @project = BasicProject.find(params[:id])
-    authorize @project
-    @project.assign_attributes(basic_project_params)
+    @basic_project = BasicProject.find(params[:id])
+    authorize @basic_project
+    @basic_project.assign_attributes(basic_project_params)
 
-    if @project.save
-      redirect_to admin_basic_project_path(@project), notice: I18n.t(:notice_updated)
+    if @basic_project.save
+      redirect_to admin_basic_project_path(@basic_project), notice: I18n.t(:notice_updated)
     else
       prep_form_vars
       render :show
     end
   end
 
   def create
-    @project = BasicProject.new(basic_project_params)
-    authorize @project
+    @basic_project = BasicProject.new(basic_project_params)
+    authorize @basic_project
 
-    if @project.save
-      redirect_to admin_basic_project_path(@project), notice: I18n.t(:notice_created)
+    if @basic_project.save
+      redirect_to admin_basic_project_path(@basic_project), notice: I18n.t(:notice_created)
     else
       prep_form_vars
       render :new
     end
   end
 
   def destroy
-    @project = BasicProject.find(params[:id])
-    authorize @project
+    @basic_project = BasicProject.find(params[:id])
+    authorize @basic_project
 
-    if @project.destroy
+    if @basic_project.destroy
       redirect_to admin_basic_projects_path, notice: I18n.t(:notice_deleted)
     else
       prep_form_vars
       render :show
     end
   end
 
+  def change_date
+    @basic_project = @basic_project = BasicProject.find(params[:id])
+    authorize @basic_project, :update?
+    attrib = params[:which_date] == "project_start" ? :signing_date : :end_date
+    @basic_project.update_attributes(attrib => params[:new_date])
+    render nothing: true
+  end
```

Comment:
How come this got added here as well as in the concern? Should be able to remove it once we have the separate controller.